demisto_sdk/commands/common/hook_validations/conf_json.py

```diff
@@ -144,3 +145,18 @@ def integration_has_unskipped_test_playbook(self, integration_data, integration_
                     test_playbook_ids.append(test['playbookID'])
 
         return self.has_unskipped_test_playbook(integration_data, integration_id, file_path, test_playbook_ids)
+
+    def get_test_path(self, file_path):
+        test_path = Path(file_path)
+        test_file_name = test_path.parts[-1].replace('.yml', '_test.py')
```

Comment:
why is the function being called with `.yml` instead of `_test.py`>

Reply:
Because this is the function that creates the test_path from the yml_path.